config/global.ini.php

```diff
@@ -552,13 +552,19 @@
 ; - "Email server settings"
 enable_general_settings_admin = 1
 
+; By setting this option to 0, all outgoing connections will be disabled
+; It also disables enable_auto_update, enable_update_communication if enabled!
```

Comment:
Actually it doesn't disable those settings - it makes them being ignored in current use cases. Maybe it's better to write something like:
`Disabling this will disable features like automatic updates for Piwik, its plugins and components like the GeoIP database, referrer spam blacklist or search engines and social network definitions`